### ISSUE TITLE
How to set Tacotron2 for Polish language?

### ISSUE THREAD

**Issue by VexorionReal** [NVIDIA/tacotron2#468]
Hey! I am new here! Do you know how to set Tacotron 2 for Polish language samples?

**Comment by 41WhiteElephants** [comment]
Yes, I have succesfully trained Polish language on Czubówna speech samples. 3-6 hours of dataset would be fine. The best way is to resume training with your polish dataset on pre-trained english model. After about 30k of iterations you should see comprehendable results. Add polish symbols to 
https://github.com/NVIDIA/tacotron2/blob/master/text/symbols.py & then you should start new training with polish dataset with flags  --checkpoint_path official_english_model.pt and --warm_start 

here samples from my polish model:
https://drive.google.com/file/d/1SvsbdRcw5bADOSyh6YM2A-6u_80veH6d/view?usp=sharing

**Comment by rave974** [comment]
> Yes, I have succesfully trained Polish language on Czubówna speech samples. 3-6 hours of dataset would be fine. The best way is to resume training with your polish dataset on pre-trained english model. After about 30k of iterations you should see comprehendable results. Add polish symbols to
> https://github.com/NVIDIA/tacotron2/blob/master/text/symbols.py & then you should start new training with polish dataset with flags --checkpoint_path official_english_model.pt and --warm_start
> 
> here samples from my polish model:
> https://drive.google.com/file/d/1SvsbdRcw5bADOSyh6YM2A-6u_80veH6d/view?usp=sharing

how did you resume training with polish dataset on pre-trained english model? after adding polish symbols to symbol.py didn't you get mismatch error? if so, how did you get pass that error?

**Comment by rave974** [comment]
followed, train seems to wotk but on inference i get error that says alphabets dont match

**Comment by 41WhiteElephants** [comment]
Hey, take a look at my fork of tacotron, it's the one I used for training with all symbols and weights adjusted for polish speech ;)
https://github.com/41WhiteElephants/tacotron2

**Comment by rave974** [comment]
works, i don't know why i had error on inference, but i reinstalled everything and followed your steps and it works. non english dataset with 3700wavs trained 100epochs with continue train from english checkpoint. inferencing using pretrained english waveglow gives perfect voice sound quality, but intonation, speed, punctuation, and some words sometimes are not good enough i guess it needs more training or a bigger dataset.

**Comment by NeonBohdan** [comment]
Very cool
@41WhiteElephants can you please tell where the Czubówna speech samples can be downloaded
